tcms/testruns/tests.py

```diff
@@ -37,7 +36,7 @@ def setUpTestData(cls):
         cls.client = test.Client()
 
     def test_404_if_run_does_not_exist(self):
-        nonexisting_run_pk = TestRun.objects.count() + 1
+        nonexisting_run_pk = 999999
```

Comment:
It would be helpful to understand if you can add explanation to this change. Then, I'll merge it.